test/verifier.test.js

```diff
@@ -95,32 +97,47 @@ describe('Verifier', () => {
           expect(result).to.deep.equal(user);
         });
       });
+
+      it('allows dot notation for password field', () => {
+        const oldPasswordField = verifier.options.passwordField;
```

Comment:
I just realized that we don't need to store and reset `oldPasswordField` since everything will be reset before every test.